### PR TITLE
[qt5] Add -j to make to parallelize on Linux and MacOS

### PR DESCRIPTION
**Pull request by BillyONeal**

### CODE REVIEW

**Comment by Neumann-A** [comment]
I think there is a VCPKG variable for concurrency. VCPKG_CONCURRENCY or similar. Should be visible with --debug in the cmake call if I am not mistaken. Currently on mobile so I cannot check it.

**Comment by jwillemsen** [comment]
There is a VCPKG_CONCURRENCY, see https://github.com/microsoft/vcpkg/blob/3a1e8e01f57ab54559a588cb4cd9442a2f27b6c8/ports/ace/portfile.cmake#L237 where we use it work regular make

File: scripts/cmake/vcpkg_build_qmake.cmake [hunk 1]
```diff
@@ -29,6 +30,7 @@ function(vcpkg_build_qmake)
 
     function(run_jom TARGETS LOG_PREFIX LOG_SUFFIX)
         message(STATUS "Package ${LOG_PREFIX}-${TARGET_TRIPLET}-${LOG_SUFFIX}")
+        list(PREPEND TARGETS ${INVOKE_ARGS})
```
Comment (Neumann-A) [inline scripts/cmake/vcpkg_build_qmake.cmake:33]: CMake too old on Linux CI. PREPEND was introduced with 3.15 https://cmake.org/cmake/help/v3.15/release/3.15.html#commands

**Comment by Neumann-A** [comment]
Since you are already touching concurrency settings:
https://github.com/microsoft/vcpkg/blob/0ab1a9e1c64a3968631b59647e771beda7d1f256/ports/qt5-base/cmake/install_qt.cmake#L6-L26
could also be removed and replaced with VCPKG_CONCURRENCY

File: scripts/cmake/vcpkg_build_qmake.cmake [hunk 2]
```diff
@@ -6,21 +6,22 @@
...
     cmake_parse_arguments(_csc "SKIP_MAKEFILES" "BUILD_LOGNAME" "TARGETS;RELEASE_TARGETS;DEBUG_TARGETS" ${ARGN})
 
     if(CMAKE_HOST_WIN32)
         vcpkg_find_acquire_program(JOM)
         set(INVOKE "${JOM}")
+        set(INVOKE_ARGS "")
     else()
         find_program(MAKE make)
         set(INVOKE "${MAKE}")
+        set(INVOKE_ARGS "-j;${VCPKG_CONCURRENCY}")
```
Comment (strega-nil) [inline scripts/cmake/vcpkg_build_qmake.cmake]:
```suggestion
        set(INVOKE_ARGS -j ${VCPKG_CONCURRENCY})
```

**Review by ras0219-msft — APPROVED** [review]

**Pull request merged**